ios/Classes/BetterPlayer.m

```diff
@@ -1009,28 +1020,48 @@ - (void) initPlayerCoverView {
 }
 
 - (void) showPlayerCoverView {
-    if (self._betterPlayerView == nil) {
```

Comment:
The previous code, checking `self._betterPlayerView == nil`.
Don't we need it before calling `self._betterPlayerView.superview`?

Reply:
@noboru-i 
No, in Objective-C, we don't need that check.

Reply:
Ah, sorry.
I understand :bow: